experiments/react-native-uifabric/src/components/RadioGroup/RadioButton.tsx

```diff
@@ -28,60 +28,57 @@ export const RadioButton = compose<IRadioButtonType>({
     // Grabs the context information from RadioGroup (currently selected button and client's onChange callback)
     const info = React.useContext(RadioGroupContext);
 
-    const pressable = useAsPressable({
-      ...rest
-    });
+    const pressable = useAsPressable(rest);
 
     const state: IRadioButtonState = {
-      ...pressable.state,
-      selected: info.selectedKey === userProps.buttonKey,
-      disabled: disabled || false
+      info: {
```

Comment:
I think you misunderstood my comment--- I suggested making this change in RadioGroup, not in RadioButton.  I.e.

... in RadioButton
```
const state: IRadioButtonState = {
   ...pressable.state,
   selected: info.selectedKey === userProps.buttonKey,
   disabled: disabled || false
}
```
... in RadioGroup
```
const state: IRadioGroupState = {
      context (or info): { 
           selectedKey: data.selectedKey,
           onChange: data.onChange
       }
    };
```
The reason for this is because of how the line below works:
`useStyling(userProps, (override: string) => state[override] || userProps[override]);`

What this does is it looks at the `state` object and says, "Hey, do I have a 'disabled' state?" and if so, applies the __overrides from 'disabled' in the settings file.  I.e.

```
disabled: {
        tokens: {
          borderColor: 'buttonBorderDisabled',
          color: 'disabledBodyText',
          backgroundColor: 'background'
        }
      }
```

since no __override corresponds to `onChange` or `selectedKey` it doesn't make sense in Radio**Group** to pass that into the override state function.  Your just putting into the `state` to smuggle additional details into the render function for the purpose of setting the RadioGroup Context (hence why I'm proposing calling it `context` instead of `info` like Button).

**Technically** this does mean if you wrote an __override called `context` in the settings file, like

`__precedence: [ context, otherstuff... ]`
`__overrides: { context: { tokens: { borderColor: 'red' } } }`

that would get applied, but I think it conveys the code's intent better

Reply:
Realistically this isn't a huge detail, I felt liked I owed you some further clarification, though :)

Reply:
Ok this makes sense, I wasn't too sure what the (override: string) => state[override] || userProps[override]) did but this clears it up for me. I'll look into changing this now

Reply:
Changed in my newest iteration